src/components/Common/MultipleSelect/MultipleSelect.stories.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import MultipleSelect from "./MultipleSelect";
+import { Option, SelectContainer } from "./MultipleSelectElements";
+
+export default {
+    title: "Components/MultipleSelect/MultipleSelect",
+    component: MultipleSelect,
+};
+
+const Template = ({ optionsValuesWithLabels, ...args }) => (
```

Comment:
![image](https://github.com/thecyberworld/TheCyberHUB/assets/76536506/30f20e81-df8a-4c56-9d80-09fd0f6332c2)

I see the label in the website, but not in the storybook

![image](https://github.com/thecyberworld/TheCyberHUB/assets/76536506/ca36d9f7-4c16-4c54-ad7f-9290e3688759)

Reply:
Could be because the label is affected by global css? and as a separate component its bot being affected?

Reply:
You right! see we already find storybook useful haha O_O
Would you like to modify the component so it will look properly?